applications/pelion_client/sample.yaml

```diff
@@ -0,0 +1,14 @@
+sample:
+  name: Pelion Client Application
+  description: Pelion conectivity reference application
+tests:
+  applications.pelion_client.zdebug:
+    build_only: true
+    build_on_all: true
```

Comment:
Please remove `build_on_all: true`. It tells Twister to try to build on all Zephyr's platforms (~400) which are then skipped and produce noise.